### PR TITLE
Add more #[\SensitiveParameter]

### PR DESCRIPTION
**Pull request by fancyweb**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

Follow-up to https://github.com/symfony/symfony/pull/46183

### CODE REVIEW

File: src/Symfony/Component/Security/Csrf/CsrfTokenManager.php [hunk 1]
```diff
@@ -79,7 +79,7 @@ public function getToken(string $tokenId): CsrfToken
         return new CsrfToken($tokenId, $this->randomize($value));
     }
 
-    public function refreshToken(#[\SensitiveParameter] string $tokenId): CsrfToken
+    public function refreshToken(string $tokenId): CsrfToken
```
Comment (fancyweb) [inline src/Symfony/Component/Security/Csrf/CsrfTokenManager.php:82]: @GromNaN I think this one was a mistake. It was not added to the interface and all other CSRF token ids. WDYT?

File: src/Symfony/Component/Security/Csrf/TokenStorage/TokenStorageInterface.php [hunk 2]
```diff
@@ -28,7 +28,7 @@ public function getToken(string $tokenId): string;
     /**
      * Stores a CSRF token.
      */
-    public function setToken(string $tokenId, string $token);
+    public function setToken(string $tokenId, #[\SensitiveParameter] string $token);
```
Comment (fancyweb) [inline src/Symfony/Component/Security/Csrf/TokenStorage/TokenStorageInterface.php:31]: From https://github.com/symfony/symfony/pull/46183:
> The attribute has no effect on the interface, but is a remainder for implementers.

**Review by GromNaN — APPROVED** [review]
Good catch.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @fancyweb.

**Pull request merged**